src/g_game.c

```diff
@@ -749,7 +749,7 @@ static void G_DoLoadLevel(void)
     headsecnode = NULL;
    }
 
-  critical = (gameaction == ga_playdemo || demorecording || D_CheckNetConnect());
+  critical = (gameaction == ga_playdemo || demorecording || demoplayback || D_CheckNetConnect());
```

Comment:
The reason I used `gameaction == ga_playdemo` here is because `demoplayback` is not set yet in this function if a single level demo is playing. Thanks, looks good to me.